src/styles/base/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { global } from "./global";
+import { typography } from "./typography";
+
+export const base = (): string => {
+  return `
+    ${global()}
```

Comment:
I think tailwind allows you to customise the base layer, not sure if this method of overwriting is better... I'm guessing the other alternative is to install the font and use the font family as default on tailwind config. wdyt?

Discussion: https://github.com/tailwindcss/discuss/issues/277

Reply:
i've set tailwind config as the base and source of truth, there is some workaround though, need to put the config within ./src folder... abit erxin